index.html

```diff
@@ -4,19 +4,19 @@
         <meta charset="UTF-8">
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <title>Portfolio</title>
+        <title>Abidemi's Portfolio Site</title>
         <link rel="stylesheet" href="style.css">
         <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet">
         <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"/>
     </head>
-    <body>
+    <body class="scroll" id="body">
 
         <!--Header-->
         
         <header class="header">
-            <a href="#" class="logo">Abidemi</a>
+            <a href="#" class="logo" id="my_name">Abidemi</a>
             <nav class="menu-icon">
```

Comment:
- [ ] It would be better if we increase the size of the hamburger bottom to achieve the figma design as mentioned earlier. Here are screenshots of how we can fix this and how it will look after fixing

This is how our hamburger looks now
![Screenshot from 2022-09-12 07-57-45](https://user-images.githubusercontent.com/46388113/189604750-290cef7e-0809-4e87-8aee-cc151a49cb4d.png)

Here is the fix

![Screenshot from 2022-09-12 07-56-55](https://user-images.githubusercontent.com/46388113/189604737-078fc7ee-3ab4-415e-93e4-cdf7e2ee1624.png)
![Screenshot from 2022-09-12 07-57-14](https://user-images.githubusercontent.com/46388113/189604741-663c7b92-4ceb-406c-80a2-cdaeb30b9b58.png)

Here is how it looks now after the fix
![Screenshot from 2022-09-12 07-56-40](https://user-images.githubusercontent.com/46388113/189604726-c1a11019-a768-4d20-9e37-9868305433d1.png)